evaluation/cli.py

```diff
@@ -97,12 +97,15 @@ def red_teaming(
         help="Path to the directory where the scorer YAML files are stored.",
         default=EVALUATION_DIR / "scorer_definitions",
     ),
-    prompt_target: Optional[str] = typer.Option(default="openai"),
+    prompt_target: Optional[str] = typer.Option(default="application"),
 ):
     red_team = service_setup.get_openai_target()
-    target = (
-        service_setup.get_openai_target() if prompt_target == "openai" else service_setup.get_azure_ml_chat_target()
-    )
+    if prompt_target == "application":
+        target = service_setup.get_app_target()
+    elif prompt_target == "azureopenai":
+        target = service_setup.get_openai_target()
+    elif prompt_target == "azureml":
+        target = service_setup.get_azure_ml_chat_target()
```

Comment:
What if it's none of these? Nowhere do we mention that this is an enumeration or that it can only take specific values. That should be exposed through the CLI, definitely as a help message. And we should be handling the case where the provided value doesn't match any of these. Not only is an error not being returned here, but you proceed with an undefined target to run the red teaming.